src/scroll-animation.js

```diff
@@ -49,6 +48,9 @@ export default class ScrollAnimation extends Component {
         clearTimeout(tid);
       })
     }
+    if (this.props.animateOnce && this.state.lastVisibility.completely) {
+      return;
```

Comment:
Wanted to call attention to this - we just exit the function if we were asked to animate only once and we have already been completely visible. Relying on `lastVisibility.partially` did not fix the issue in my experience, but this has resolved it for me.